contrib/runners/pythonrunner/pythonrunner.py

```diff
@@ -0,0 +1,247 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
Another thing - let's make sure that those files are still linted (might need to update lint targets in the makefile, but we already lint some other Python files in that directory, might just need to add contrib/runners/*/.py to it or similar).